Week1/exercise_1.js

```diff
@@ -0,0 +1,76 @@
+const mysql = require('mysql');
+
+const connection = mysql.createConnection({
+    host: 'localhost',
+    user: 'hyfuser',
+    password: 'hyfpassword',
+    multipleStatements: true
+});
+
+
+/*Connect to SQL server*/
+connection.connect(err => {
+    if (err) {
+        return console.error('Connection error: ' + err.stack);
+    }
+    console.log('Connected!');
+});
+
+
+/*SQL queries*/
+const createDatabaseAndTables =
+    `DROP DATABASE IF EXISTS meetup;
```

Comment:
Proposing to move `meetup` to a const. Like `const DB_NAME = 'meetup'` and use this in the template string. This way you can avoid typos.